10/chapter-06.srt

```diff
@@ -224,6 +224,9 @@ So I will print current stanza here.
 00:04:19,330 --> 00:04:25,290
 And maybe the reps as well so we can check it.
 
+
+4:25 - Drew: Sure.
```

Comment:
Don't know in which way new sentences are injected, but here I think we need this.